src/triage/component/architect/builders.py

```diff
@@ -153,7 +156,7 @@ def make_entity_date_table(
         )
         logging.info('Creating matrix-specific entity-date table for matrix '
                      '%s with query %s', matrix_uuid, query)
-        self.engine.execute(query)
+        self.db_engine.execute(query)
```

Comment:
Is there still any need for these changes to `engine` and `sessionmaker`?

Reply:
Well the change from `engine` to `db_engine` is just for consistency with other components. It's not necessary, but I think still helps.

I did try moving the sessionmaker back to the constructor. It does cause serialization issues, even with a serializable engine. I could, of course, apply the same pattern to make a serializable sessionmaker, but this isn't nearly as compelling as it would have to be copied to all the components that need one (whereas db_engine being passed in from the caller only has to be done in one place)

Reply:
Sure, makes sense :+1: 

At some point we could make it a cached property (and ignored on pickling), but that's hardly a priority.